packages/eas-cli/src/commandUtils/workflow/utils.ts

```diff
@@ -0,0 +1,110 @@
+import { fetchRawLogsForJobAsync } from './fetchLogs';
+import {
+  WorkflowJobResult,
+  WorkflowLogLine,
+  WorkflowLogs,
+  WorkflowRunResult,
+  WorkflowTriggerType,
+} from './types';
+import { WorkflowRunFragment, WorkflowRunStatus } from '../../graphql/generated';
+import { Choice } from '../../prompts';
+
+export function computeTriggerInfoForWorkflowRun(run: WorkflowRunFragment): {
```

Comment:
Would https://github.com/expo/universe/blob/d3a21acde7564bc652704337be8f979f7b8d5d3a/server/www/src/graphql/types/WorkflowRun.ts#L40 be helpful?

Reply:
Yes I’ll pull the trigger type definition into this PR

Reply:
Done